moto/cloudformation/responses.py

```diff
@@ -250,6 +294,17 @@ def list_exports(self):
 </UpdateStackResponse>
 """
 
+CREATE_CHANGE_SET_RESPONSE_TEMPLATE = """<CreateStackResponse>
+  <CreateChangeSetResult>
+    <Id>{{change_set_id}}</Id>
+    <StackId>{{ stack_id }}</StackId>
+  </CreateChangeSetResult>
+  <ResponseMetadata>
+    <RequestId>b9b4b068-3a41-11e5-94eb-example</RequestId>
```

Comment:
Look [here](https://github.com/spulec/moto/blob/81f3cbb54880bdf68fd6f52b9aa30ea25a4b7bd9/moto/cloudwatch/responses.py#L18) and [here](https://github.com/spulec/moto/blob/81f3cbb54880bdf68fd6f52b9aa30ea25a4b7bd9/moto/cloudwatch/responses.py#L195) for returning a proper request ID. If your up to it, see if the other methods can be done too

Reply:
Cool - thanks - will take a look at it!